compiler/rustc_lint/src/lib.rs

```diff
@@ -640,6 +640,11 @@ fn register_builtins(store: &mut LintStore) {
          see <https://github.com/rust-lang/rust/issues/40107> for more information",
     );
     store.register_removed("wasm_c_abi", "the wasm C ABI has been fixed");
+    store.register_removed(
+        "uninhabited_statics",
```

Comment:
```suggestion
        "uninhabited_static",
```

The lint was (mis)named `uninhabited_static`.  Some of the crater breakage is due to this and people having set warnings to be denied.